src/main/resources/static/js/global-variables.js

```diff
@@ -106,20 +122,4 @@ function getWhiteListedPages() {
     return String(PAGES_FOR_STATUS_INDICATION);
 }
 
-function getSubscriptionNameRegex() {
-    return subscriptionNameRegex;
-}
-
-function getNotificationMetaRegex() {
-    return notificationMetaRegex;
-}
-
-function getUsernameMaxDisplayLength() {
-    if (usernameMaxDisplayLength < 4) {
-        // Minimum leangth is 4 due to ellipsis being 3 long
-        return 4;
-    }
-    return usernameMaxDisplayLength;
-}
-
-// End   ## getters and setters status handling
\ No newline at end of file
+// End   ## getters and setters status handling
```

Comment:
I think "getters and setters" should be removed from comment. It is more confusing than not to have it here in my opinion.

Reply:
I don't think the of the divider comments is necessary. I would prefer it if they are removed altogether